definitions/npm/reselect_v3.x.x/flow_v0.104.x-/reselect_v3.x.x.js

```diff
@@ -4,14 +4,12 @@ declare module "reselect" {
   declare type InputSelector<-TState, TProps, TResult> =
     (state: TState, props: TProps, ...rest: any[]) => TResult
 
```

Comment:
I was trying to add a test for the code which fails in our code base, but somehow it does not show the error when put in this file. However, the change fixes the error in our code, same as PR for v4.